PointsRequest request
---
std_msgs/Float64MultiArray points